src/bluesearch/entrypoint/database/add_es.py

```diff
@@ -115,12 +115,25 @@ def bulk_paragraphs(
     for inp in inputs:
         serialized = inp.read_text("utf-8")
         article = Article.from_json(serialized)
+        # add abstract to paragraphs in order to be able to search for abstracts
+        for i, abstract in enumerate(article.abstract):
+            doc = {
+                "_index": "paragraphs",
+                "_source": {
+                    "article_id": article.uid,
+                    "section": "abstract",
+                    "text": abstract,
+                    "paragraph_id": i,
+                },
+            }
+            yield doc
+        # add body paragraphs
         for ppos, (section, text) in enumerate(article.section_paragraphs):
             doc = {
                 "_index": "paragraphs",
                 "_source": {
                     "article_id": article.uid,
-                    "section_name": section,
+                    "section": section,
                     "text": text,
                     "paragraph_id": ppos,
```

Comment:
If I remember correctly, the goal of this `paragraph_id` is to hold a unique number (just an increment integer) for each paragraph. However, if I am not mistaken after introduction of the abstract paragraphs into the paragraphs table we might have collisions

Reply:
I agree, maybe we could put `"paragraph_id": ppos + i + 1` to avoid any collisions. What do you think @drsantos89 @jankrepl ?